### ISSUE TITLE
"Logged in to the wrong region. Switch to 'EU2' region"

### ISSUE THREAD

**Issue by 0x5f3** [timoschlueter/nightscout-librelink-up#97]
```
[info]: renew token
[error]: LibreLink Up - Logged in to the wrong region. Switch to 'EU2' region.
[error]: LibreLink Up - No AuthTicket received. Please check your credentials.

```

Changing the region to EU2, the error is still showing

I presume it requires adding the new api endpoint to: https://github.com/timoschlueter/nightscout-librelink-up/blob/main/src/constants/llu-api-endpoints.ts

**Comment by timoschlueter** [comment]
Looks like they are adding regions. I will look into this as soon as today!

**Comment by timoschlueter** [comment]
Fixed the problem with v2.3.0. I hope it works for you now :)
